### PR TITLE
MOU signing by users

### PR DESCRIPTION
**Pull request by thomasiles**
### What problem does this pull request solve?

Trello card: https://trello.com/c/q0Ud4qRd/1084-mou-digital-page-and-process-that-fires-data-to-the-database

Allow users to sign MOUs.

This PR uses placeholder for the text of the MOU as that will come in another PR.

### Things to consider when reviewing

- Ensure that you consider the wider context.
- Does it work when run on your machine?
- Is it clear what the code is doing?
- Do the commit messages explain why the changes were made?
- Are there all the unit tests needed?
- Has all relevant documentation been updated?

### CODE REVIEW

**Review by DavidBiddle — APPROVED** [review]
Looks good! 🎉

File: app/controllers/mou_signatures_controller.rb [hunk 1]
```diff
@@ -1,9 +1,48 @@
 class MouSignaturesController < ApplicationController
-  after_action :verify_authorized
+  after_action :verify_authorized, only: %i[index]
```
Comment (aliuk2012) [inline app/controllers/mou_signatures_controller.rb:2]: If none of the other actions need authorization then its ok to remove this before action.

Comment (thomasiles) [reply app/controllers/mou_signatures_controller.rb:2]: as in just delete it?

Comment (thomasiles) [reply app/controllers/mou_signatures_controller.rb:2]: I'm going to merge this as it is and tackle this in a follow-up PR if thats ok. I have another small issue I need to fix so I'll throw it in with that.

**Pull request merged**